upgrade/breaking-changes/11-0-0.md

```diff
@@ -10,6 +10,26 @@ position: 7
 
 This article lists the breaking changes introduced with the Telerik UI for .NET MAUI 11.0.0 release version.
 
+## Common Changes
+
+The `SkiaSharp` is updated to `3.116.1` version.
+
+Trial and commercial users now use the same product package. The trial and commercial product versions merged into a single unified distribution package. The product access now depends on a license key file. This eliminates the need for separate trial downloads. For more information, please refer to the [Set up your license]({%slug set-up-your-license%}) documentation article.
```

Comment:
Note the product the users are using. Without context, it's easy to assume you're continuing the theme from the previous paragraph about SkiaSharp, rather than Telerik UI for .NET MAUI.